sql/sys_vars.inl

```diff
@@ -795,6 +795,51 @@ protected:
   bool set_filter_value(const char *value, Master_info *mi);
 };
 
+class Sys_var_binlog_filter: public sys_var
+{
+private:
+  int opt_id;
+  privilege_t m_access_global;
+
+public:
+  Sys_var_binlog_filter(const char *name, int getopt_id, const char *comment,
+                     privilege_t access_global)
+    : sys_var(&all_sys_vars, name, comment, sys_var::GLOBAL, 0, NO_GETOPT,
```

Comment:
Just a couple other notes, the `flags_arg` will need to be `sys_var::READONLY+sys_var::GLOBAL`. Then below, we don't need `GET_ASK_ADDR`, just `GET_STR` should suffice.